### PR TITLE
Remove SPO list

### PR DESCRIPTION
**Pull request by rdlrt**
It would be fundamentally wrong to try and add pools subset list on essentials (you have over 2000 pools on chain, out of which atleast 60-70% can be considered 'active', and listing it on this page does not necessarily mean they're actually active - and represents a very wrong picture + advertisement for ecosystem).

### CODE REVIEW

**Comment by adrem1** [comment]
Hi @rdlrt,

I was a bit surprised to see that list here in the essentials, nonetheless added my pool and a couple of mates'. Whereby I agree with you regarding the number of non-represented pools, I don't feel so strongly about the negative effects on the outlook for the ecosystem. After all, without SPOs the network in its current state would not exist. Further, a number of issues have been raised about the "decentralisation" effect that current ranking systems have on small pools (https://forum.cardano.org/t/cip-stake-uri-scheme-for-pools-delegation-portfolios/40594/36)

Albeit not very practical, a curated list of pools should exist in an official document such as the essentials, not just in third party tools, aggregators and wallets. That's just my opinion. **Would it be fairer if all the pools had their names here?** 

It would be quite some work and would probably require a separate page/section all together. Having said that an initial screen for pools that don't meet criteria for active status (eg not registered properly, pledge not met, etc) would surely bring the number down considerably from the 2000 mark. If at some point a consensus is reached in regards to this matter, I would be happy to help add more pools to the list until they are all there!

What do you think?

Cheers,

Stef

**Comment by rdlrt** [comment]
@adrem1 - To me readability, accessibility and usability of page will take a hit trying to post all active pools (easily over 1000). You already have information about exploring pools on the page, feels quite unnecessary to make it another point of advertisement/competition (which is found in abundance in community)

File: essential-cardano-list.md [hunk 1]
```diff
@@ -467,47 +467,3 @@ Tools to help you build on Cardano:
 
 ### Other IOHK Solutions ###
 - [Mantis](https://mantisclient.io/)
-
-### 🥩 Stake Pools 🥩 ###
```
Comment (kiriakos) [inline essential-cardano-list.md:471]: Maybe we could replace this section with a short "finding a stake pool" section?

**Review by kiriakos — APPROVED** [review]

File: essential-cardano-list.md (hunk 1 above)
Comment (rdlrt) [reply essential-cardano-list.md:471]: We have a [exploring pools](https://github.com/input-output-hk/essential-cardano/blob/main/essential-cardano-list.md#exploring-stake-pools) section already on this page linking to various ways to find stake pool from different websites, but yes - happy for that section to be magnified instead

**Comment by nahern** [comment]
thanks for your feedback, we decided to include the stake pool list for now, aware that of course it could grow to 2k+ pools over time. However, there is value to its inclusion. We will be monitoring this and watching how large it grows, if readability is affected I plan to move this to an appendix in the list, off the main page.

**Comment by AndrewWestberg** [comment]
It's unreasonable for pool operators to maintain a github page for pools when all the information is available on the blockchain and already aggregated by many good community tools. Not every pool operator is well-versed in github.

@nahern , you should revisit your decision. See -> https://github.com/input-output-hk/essential-cardano/pull/40

**Comment by adosiawolf** [comment]
unreasonable is being generous - it's absurd to expect stake pool operators to maintain this data when it already readily exists on-chain

**Pull request closed without merging**